templates/resources.hbs

```diff
@@ -26,7 +26,7 @@
 	<div class="resources-item-container">
 	{{#each resources}}
 	<div class="resources-item drawer-item {{_type}}">
-		<button class="base resources-item-open drawer-item-open" data-type="{{_type}}" data-filename="{{filename}}" data-href="{{_link}}" tabindex="0" role="button" aria-label="{{lookup ../model._filterButtons _type}}. {{../model.itemAriaExternal}}. {{{title}}}. {{{description}}}">
+		<button class="base resources-item-open drawer-item-open" data-type="{{_type}}" data-filename="{{filename}}" data-href="{{_link}}" data-forceDownload="{{forceDownload}}" tabindex="0" role="button" aria-label="{{lookup ../model._filterButtons _type}}. {{../model.itemAriaExternal}}. {{{title}}}. {{{description}}}">
```

Comment:
could you lowercase `data-forceDownload` to match the JS code? I know it works as is but I think it should be consistent